tasks/test.yaml

```
@@ -226,3 +228,29 @@ tasks:
       - task: test-resources:e2e-test
         with:
           validate_egress: "false"
+
+  - name: uds-core-private-pki
+    description: "Create and Deploy Private PKI Bundle"
+    actions:
+      - task: playwright:private-pki-certs
```

very nit related to above comment
```suggestion
      - task: private-pki:private-pki-certs
```